### DISCUSSION TITLE
Summany column does not work

### DISCUSSION THREAD

**Discussion by daniieljc** [Power-Components/livewire-powergrid#1786] [Q&A]
Column summany does not work, it appears as if it is activated but does not show data inside it.

Should I add something? I am using the latest version, v6.1.6

`Column::make('Payout', 'payout')->withSum('Sum Price', header: true, footer: false),
`

**Comment by eafarooqi** [comment]
fixed #1802

**Reply by daniieljc** [reply]
Thank you for your input.

Could you take a look at this other one?

https://github.com/Power-Components/livewire-powergrid/discussions/1785
